test/test_enforcer.cpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "pch.h"
+
+#include <direct.h>
+#include <algorithm>
+
+#include <enforcer.h>
+#include <persist.h>
+#include <rbac.h>
+#include <util.h>
+
+using namespace std;
+
+namespace test_enforcer
+{
+    TEST_CLASS(TestEnforcer)
+    {
+    public:
+
+        string filePath(string filepath) {
+            char* root = _getcwd(NULL, 0);
```

Comment:
A little bit wired at here, since the test execution directory is determined, that is `x64/Debug`, use a relative path is enough. Not a big deal but would like to fix this in the following commits, I found this function exists also in other files.